spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1473,9 +1503,371 @@ static void queryWithQueryOptions(DatabaseClient dbClient) {
   }
   // [END spanner_query_with_query_options]
 
+  // [START spanner_create_backup]
+  static void createBackup(
+      DatabaseAdminClient dbAdminClient, DatabaseId databaseId) {
+    String backupName =
+        String.format(
+            "%s_%02d",
+            databaseId.getDatabase(), LocalDate.now().get(ChronoField.ALIGNED_WEEK_OF_YEAR));
+    BackupId backupId = BackupId.of(databaseId.getInstanceId(), backupName);
+    // Set expire time to 14 days from now.
+    long currentTimeInMicroSeconds =
+        TimeUnit.MICROSECONDS.convert(System.currentTimeMillis(), TimeUnit.MILLISECONDS);
+    long deltaTimeInMicroseconds = TimeUnit.MICROSECONDS.convert(14L, TimeUnit.DAYS);
+    Timestamp expireTime =
+        Timestamp.ofTimeMicroseconds(currentTimeInMicroSeconds + deltaTimeInMicroseconds);
+    Backup backup =
+        dbAdminClient
+            .newBackupBuilder(backupId)
+            .setDatabase(databaseId)
+            .setExpireTime(expireTime)
+            .build();
+    // Initiate the request which returns an OperationFuture.
+    System.out.println("Creating backup [" + backup.getId() + "]...");
+    OperationFuture<Backup, CreateBackupMetadata> op = backup.create();
+    try {
+      // Wait for the backup operation to complete.
+      backup = op.get();
+      System.out.println("Created backup [" + backup.getId() + "]");
+    } catch (ExecutionException e) {
+      // If the operation failed during execution, expose the cause.
+      throw (SpannerException) e.getCause();
+    } catch (InterruptedException e) {
+      // Throw when a thread is waiting, sleeping, or otherwise occupied,
+      // and the thread is interrupted, either before or during the activity.
```

Comment:
I realize we have this type of comment in the `createDatabase` sample but I'm wondering if we could get rid of it here to keep things concise.

Reply:
agree, I removed it.